app/assets/scripts/components/card.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { PropTypes as T } from 'prop-types';
+import styled, { css } from 'styled-components';
+
+const CardWrapper = styled.article`
+  ${({gridColumn, gridRow}) => css`
+    grid-column: ${gridColumn || 'auto'};
+    grid-row: ${gridRow || 'auto'};
+  `}
+`;
+
+const CardContents = styled.a`
```

Comment:
Not sure if I understand the reasoning behind these empty styled-components?

Reply:
the reasoning was in case we need to override / augment any of the sass styles, i thought it's easier / more modern to do that in styled comps? also for readability?

can also see the argument that it's making things more complicated to introduce styled component styling on top of the sass. what do you think?

Reply:
Migrating to styled-components makes sense to me, in general. I meant to question only the empty ones - however I am fine with leaving them as is.